uiowa/uiowa
# [Commencement] Event theming.

tracks https://github.com/uiowa/uiowa/issues/7592

## To test

- Pull, sync commencement
- Poke around on some events and check for the following things:  

### Two column main content
- Body field properly renders
- Contact name, email, and phone properly render

### Two column sidebar card
-  Presidential seal outputs in the media slot in the sidebar.
- Date is output in sidebar - start time & end time - there is a note about times are in Central Time.
- Venue is output in sidebar with room when there is one.
- 'Program' is displayed in sidebar without pdf icon and links to the program for the selected session.  
    - (Note: you will have to go to the session taxonomy term referenced by the event and add a session PDF there for it to show up on the rendered node)
-  'Order of events' is displayed in sidebar without pdf icon.
- The Website field renders in the sidebar
    - (Note: I was unable to find any ceremony with a website, so I think this works based upon the manage display page for the layout of the event content type)
-  There is a 'View on the events site' button that links to the event's page on events.uiowa.edu.

### Livestream

- The livestream section is on a gray background and has a heading that can be linked to with `#livestream`. Additionally, it should be properly structured in the order of headings (eg. h1, h2, h3, etc).
- Livestream field properly renders

### Livestream

- The Ceremony information section has a heading that can be linked to with `#ceremony-information`. Additionally, it should be properly structured in the order of headings (eg. h1, h2, h3, etc).
- Livestream field properly renders
- Ceremony information properly renders
- Ceremony information details properly renders
